src/vs/platform/gpu/common/gpuStatusMain.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { createDecorator } from '../../instantiation/common/instantiation.js';
+
+export const IGPUStatusMainService = createDecorator<IGPUStatusMainService>('gpuStatusService');
+
+export interface IGPUStatusMainService {
+	readonly _serviceBrand: undefined;
+	openGPUStatusWindow: (options: OpenGPUStatusWindowOptions) => void;
+}
+
+export interface OpenGPUStatusWindowOptions {
```

Comment:
nit: not blocking, but seems a little strange to me to only have a single option in this interface. are we considering adding more options?